service-front/module/Application/view/application/general/home/accessibility.twig

```diff
@@ -37,7 +37,7 @@
     </ul>
     <h2 class="heading-medium">What to do if you cannot access parts of this service</h2>
     <p>If you cannot use this service, you can use the <a href="https://www.gov.uk/government/publications/make-a-lasting-power-of-attorney">paper version of the LPA</a>. </p>
-    <p>You can email <a href="mailto:customerservices@publicguardian.gov.uk" target="_blank">customerservices@<wbr>publicguardian.gov.uk</a> to get the paper LPA in an alternative format, such as:</p>
+    <p>You can email <a href="mailto:customerservices@publicguardian.gov.uk">customerservices@<wbr>publicguardian.gov.uk</a> to get the paper LPA in an alternative format, such as:</p>
```

Comment:
I see you removed a target=blank from this mailto.  Are we definitely getting rid of all of those then?  I thought there was some reason why these might still be needed, but maybe not anymore?

Reply:
(See my answer to your next question)